apps/user/src/store/user/user.ts

```diff
@@ -10,4 +10,10 @@ const userAtomState = atom<User>({
     },
 });
 
+const isSaveFormLoadedAtomState = atom({
+    key: 'save form loaded state',
```

Comment:
```suggestion
    key: 'load-save-form',
```
은 어떨까요? 컨벤션을 따른다면 이게 더 좋을것같아요!

Reply:
아아 알겠습니다

Reply:
form 폴더에 만들라구 한 이유는 저장된 원서를 불러오는것이기 때문에 user보단 form이 더 어울린다고 생각하였습니다 또한 form을 load하는것이고 부모에 form폴더라 명시해놨으니 load.ts가 좋을것같아요!